snapshot/snapshot.go

```diff
@@ -716,7 +718,25 @@ func overlayMount(options []string) []mount.Mount {
 		},
 	}
 }
+func (o *snapshotter) passImageNameMounts(ctx context.Context, parent string) ([]mount.Mount, error) {
+	_, pInfo, _, err := snapshot.GetSnapshotInfo(ctx, o.ms, parent)
+	if err != nil {
+		return nil, errors.Wrapf(err, "passImageNameMounts get snapshot %q info failed", parent)
+	}
+	passImageOption := fmt.Sprintf("imageurl=%s", pInfo.Labels["containerd.io/snapshot/cri.image-ref"])
 
+	var overlayOptions []string
+	overlayOptions = append(overlayOptions, passImageOption)
+
+	return []mount.Mount{
+		{
+			Type:    "fuse.nydus-overlayfs",
```

Comment:
The filesystem or mount type `fuse.nydus-overlayfs` should only appear when nydus-snapshotter has enable_nydus_overlayfs enabled in its toml config.
Moreover, we should follow the what we do in `remoteMountWithExtraOptions` like encoding the mount options.

```
	// base64 to filter easily in `nydus-overlayfs`
	opt := fmt.Sprintf("extraoption=%s", base64.StdEncoding.EncodeToString(no))
	overlayOptions = append(overlayOptions, opt)
```

Reply:
Thanks, @changweige . I have fixed it.